docs/tutorial/online-offline-events.md

```diff
@@ -1,7 +1,13 @@
 # Online/Offline Event Detection
 
 Online and offline event detection can be implemented in the renderer process
-using standard HTML5 APIs, as shown in the following example.
+using standard HTML5 NavigatorOnLine API.
```

Comment:
This line would read better as:
> using the `navigator.onLine`attribute.

Reply:
Ah, I should've noticed that. Thanks a ton!